malaphor/malaphor.py

```diff
@@ -17,11 +19,15 @@
 else:
 	input_idiom = sys.stdin.read()
 
+if debug:
+    n = 1 #Number of words it will replace
```

Comment:
Never really used, it was more of a future idea to have a control on the number of words it will replace

Reply:
Okay.